src/InstrumentationEngine.Lib/LoggerService.cpp

```diff
@@ -359,7 +357,8 @@ HRESULT CLoggerService::RecalculateLoggingFlags()
         effectiveFlags = static_cast<LoggingFlags>(effectiveFlags | sinkFlags);
     }
 
-    m_effectiveFlags = effectiveFlags;
+    //m_effectiveFlags = effectiveFlags;
```

Comment:
remove comment

Reply:
Thanks

---
In reply to: [421832135](https://github.com/microsoft/CLRInstrumentationEngine/pull/258#discussion_r421832135) [](ancestors = 421832135)